Composer/packages/client/src/components/ErrorBoundary.tsx

```diff
@@ -57,6 +57,11 @@ export class ErrorBoundary extends Component<ErrorBoundaryProps, ErrorBoundarySt
   }
 
   onErrorHandler(message, source, lineno, colno, error) {
+    const patternsToIgnore = [/ResizeObserver/g];
+    if (patternsToIgnore.some((regex) => regex.test(message))) {
+      // skip the console and application-level steps and just say we handled it
+      return true;
+    }
     console.error({ message, source, lineno, colno, error });
```

Comment:
We should log the error to the console. Move the `console.log` before the if-statement. We don't want to hide errors from the developer.